### PR TITLE
[INS-3759] [Activities] Refactoring BE part of actives for the new UI…

### PR DESCRIPTION
**Pull request by RockMother**
…. (#376)

### CODE REVIEW

File: sell/sell-api.yaml [hunk 1]
```diff
@@ -167,7 +167,7 @@ paths:
             type: string
         - name: type
           in: query
-          description: The type of activity. Is one of `USER` or `EVENT`. Activities can be system-generated (events) or created by user.
+          description: The type of activity. Is one of `USER`, `EVENT` or 'MESSAGE'. Activities can be system-generated (events) or created by user.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:170]:
```suggestion
          description: The type of activity. Is one of `USER`, `EVENT` or `MESSAGE`.  The `EVENT` activity is system-generated (e.g., when a company is created), the `USER` activities are initialized by an Inperium user (e.g., a new task), the `MESSAGE` activity covers all communication through the Message Center.
```

**Comment by olyavertwist** [comment]
Please update the ActivtyTypes tag description as well:

 - name: ActivityTypes
    description: |
      Depending on the type, activities are either generated automatically or created by users. `EVENT`-based activities (such as when you add a new company or delete a contact) or `MESSAGE` activities (email activity in the Message Center) are logged automatically. The`User`-based activities are added to the timeline by users themselves, for example, when users add notes and tasks.

**Review by olyavertwist — CHANGES_REQUESTED** [review]
I prepared a more detailed description of activity types.

File: sell/sell-api.yaml [hunk 2]
```diff
@@ -4272,6 +4273,16 @@ components:
           required:
             - eventType
             - activityDate
+    MessageActivity:
```
Comment (ShakirovaE) [inline sell/sell-api.yaml:4276]: Comment for string 4266.
Do wee need to remove "MESSAGE_SENT" from enum?

Comment (RockMother) [reply sell/sell-api.yaml:4276]: Good point.

**Review by ShakirovaE — APPROVED** [review]

File: sell/sell-api.yaml [hunk 3]
```diff
@@ -20,7 +20,8 @@ tags:
...
   - name: ActivityTypes
     description: |
-      Depending on the type, activities are either generated automatically or created by users. Event-based activities (the EVENT activity type) are logged when you add a new company or delete a contact. User-based activities ( the USER activity type) are added to the timeline by users themselves, for example, when users add notes and tasks.
+      description: |
+      Depending on the type, activities are either generated automatically or created by users. EVENT-based activities (such as when you add a new company or delete a contact) or MESSAGE activities (email activity in the Message Center) are logged automatically. TheUSER-based activities are added to the timeline by users themselves, for example, when users add notes and tasks.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:24]:
```suggestion
      Depending on the type, activities are either generated automatically or created by users. The `EVENT`-based activities (such as when you add a new company or delete a contact) or the `MESSAGE` activities (email activity in the Message Center) are logged automatically. The `USER`-based activities are added to the timeline by users themselves, for example, when users add notes and tasks.
```

**Review by olyavertwist — APPROVED** [review]

**Pull request merged**